docs/_case-studies/redfin.md

```diff
@@ -23,8 +23,40 @@ excerpt_separator: <!--more-->
    limitations under the License.
 -->
 
-Testing the excerpt
+Realtime Notifications
 
 <!--more-->
 
-Markdown content goes here
\ No newline at end of file
+Redfin is a leading full-service real estate brokerage that uses modern technology 
+to help people buy and sell homes. Notification is the critical feature to 
+communicate with Redfin’s customers, notification includes recommendations, instant 
+emails, scheduled digests and push notifications. Thousands of emails are delivered 
+to customers every minute at peak. 
+
+The notification system used to be a monolithic system, which served the company 
+well. However, as business grew and requirements evolved, it became harder and 
+harder to maintain and scale. The engineering team at Redfin decided to replace 
+the existing system with Samza primarily for Samza’s performance, scalability, 
+support for stateful processing and Kafka-integration. A multi-stage stream 
+processing pipeline was developed. Listings are first matched with customers and 
```

Comment:
Any chance we can provide a diagram from their slides and add more technical details?